k8s/aat/common/family-public-law/fpl-case-service.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: fpl-case-service
-    version: 1.11.4
+    version: 1.11.7
```

Comment:
should this not be .8? same in others as well

Reply:
oh, its moved to 8 now..it was 7 when i created PR :)

Reply:
This PR is not going to be merged until after expert UI is live. So i guess i will make this change later.

Reply:
it basically should match version in your fpl-ccd-configuration Chart.yaml which currently is .8